server/Main.hs

```diff
@@ -110,6 +111,9 @@ buildMakeActions codegenRef =
   writeCacheDb :: Cache.CacheDb -> Make.Make ()
   writeCacheDb _ = pure ()
 
+  writePackageJson :: Make.Make ()
```

Comment:
Just curious why this was added? It looks like it's not used anywhere, but I'm likely missing something (ie perhaps it is invoked externally). Just wanted to make sure it's not cruft from an earlier iteration.